client/commonFramework/show-completion.js

```diff
@@ -6,14 +6,13 @@ window.common = (function(global) {
   } = global;
 
   common.showCompletion = function showCompletion() {
-    var time = Math.floor(Date.now() - common.started);
-
     ga(
       'send',
       'event',
-      'Challenge',
-      'solved',
-      common.challengeName + ', Time: ' + time + ', Attempts: ' + 0
+      'Challenge', //category
+      'solved',    //action
+      common.challengeName + ' Attempts: ', //label
```

Comment:
To be honest with you... I don't know: the original one didn't show that neither, so...  but why? I am waiting for a meeting with tera today: I can ask him.